main.py

```
@@ -26,10 +28,16 @@ def process_query(message, history):
     if len(message['files']) == 0 :
         yield "Processing Query"
     else :
-        if message['text'].strip() == "" :
-            yield "Processing PDF"
-        else :
-            yield "waiting"
+        try:
+            for file in message['files']:
+                docs = file_processor(file)
```

Ruff (0.12.2)

Local variable `docs` is assigned to but never used

Remove assignment to unused variable `docs`

(F841)